media/CfgEditor/index.js

```diff
@@ -347,35 +347,39 @@ function registerOptimizeOptions() {
   });
 
   for (const optName in oneOptimizationList) {
-    let row = document.createElement("vscode-data-grid-row");
-
-    let cellSwitch = document.createElement("vscode-data-grid-cell");
-    let checkbox = document.createElement("vscode-checkbox");
-    checkbox.setAttribute("id", "checkboxOptimize" + optName);
-    cellSwitch.appendChild(checkbox);
-    cellSwitch.setAttribute("grid-column", "1");
-    row.appendChild(cellSwitch);
-
-    let cellName = document.createElement("vscode-data-grid-cell");
-    cellName.textContent = optName;
-    cellName.setAttribute("grid-column", "2");
-    row.appendChild(cellName);
-
-    let cellDescription = document.createElement("vscode-data-grid-cell");
-    cellDescription.textContent = oneOptimizationList[optName].description;
-    cellDescription.setAttribute("grid-column", "3");
-    row.appendChild(cellDescription);
-
-    basicOptimizeTable.appendChild(row);
+    if (Object.prototype.hasOwnProperty.call(oneOptimizationList, optName)) {
```

Comment:
I don't know which one you prefer, but I suggest that you can reduce the indentation and improve readability like below.

```suggestion
if (!Object.prototype.hasOwnProperty.call(oneOptimizationList, optName)) {
  continue;
}
```

Reply:
I agree :-D Thanks for the suggestion 😄

Reply:
Oh, I missed to clean up the braces and code indentation. sorry 🥲